src/components/test-results/TestResults.test.tsx

```diff
@@ -0,0 +1,144 @@
+import * as React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import TestResults, { ITestResultsProps } from "./TestResults";
+
+const onRepeatMock = jest.fn();
+const onRepeatWrongAnswersMock = jest.fn();
+
+const wrapper = (props: ITestResultsProps) =>
+  render(<TestResults {...props} />);
+
+beforeEach(() => jest.clearAllMocks());
+
+test("should render component and perform basic validations", () => {
+  const props: ITestResultsProps = {
+    gaTestName: "mock",
+    rightAnswers: 2,
+    wrongAnswers: 4,
+  };
+  const component = wrapper(props);
+
+  expect(component.container.querySelector(".test-results--good")).toBeNull();
+
+  expect(
+    component.container.querySelector(".test-results__title")?.textContent
+  ).toEqual("Test results");
```

Comment:
Just an FYI - there is a `toHaveTextContent` matcher that is much more useful for this kind of check :)

https://github.com/testing-library/jest-dom#tohavetextcontent